pwa.config.js

```diff
@@ -1,6 +1,7 @@
-exports.babel = function(config) {
-	config.plugins.push([
-		'styled-components',
-		{ ssr: true, displayName: true, preprocess: false },
-	]);
-};
+exports.babel = function (config) {
+	config.plugins.push(
+		['babel-plugin-styled-components', {
+			pure: true,
```

Comment:
You'll want to take these changes too @sapegin 

https://styled-components.com/docs/tooling#dead-code-elimination

Reply:
Do we even use a minifier here? Who's going to read these comments?

Reply:
Yes, both Babel and Terser react to the inline comment. Separately, this annotation type also works within Rollup